packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/verbs.js

```diff
@@ -208,6 +208,42 @@ describe('plugin-conversation', function () {
         .then((reason) => assert.equal(reason.status, 403)));
     });
 
+    describe('#post() with error handling', () => {
+      let message, convo;
+
+      beforeEach(() => {
+        message = 'testing';
+        if (convo) {
+          return Promise.resolve();
+        }
+
+        return webex.internal.conversation.create({participants})
+          .then((c) => { convo = c; });
+      });
+
+      it('post a plain text when encryption key need to rotate', () => {
+        const {defaultActivityEncryptionKeyUrl} = convo;
+
+        webex.request({
+          method: 'GET',
+          api: 'conversation',
+          resource: `/conversations/${convo.id}/healed?resetKey=true`
+        })
+          .then(() => webex.request({
+            method: 'GET',
+            api: 'conversation',
+            resource: `conversations/${convo.id}`
+          }))
+          .then((res) => {
```

Comment:
I think this should be fine, as healed doesn't pass anything to the next request.
And this still passes locally

```suggestion
Promise.all([
	webex.request({
       	method: 'GET',
		api: 'conversation',
		resource: `/conversations/${convo.id}/healed?resetKey=true`
	}),
	webex.request({
        method: 'GET',
        api: 'conversation',
        resource: `conversations/${convo.id}`
    })
]).then(([, res]) => {
```

Reply:
does putting it on promise make sure that get convo called later after heal?

Reply:
So this is the first time i've seen healed and, off the top of my head, don't know where it's documented to see what it does (can send me the link on teams).

Reply:
`Promise.all` runs them in async and thens `then` after all promises are resolved, so if it needs to be one after the other, i'm fine with the current way its written.
I just don't really like nested requests, but i dont know a cleaner way of writing them besides the `.all`